Sources/SafeDICore/Generators/DependencyTreeGenerator.swift

```diff
@@ -105,7 +105,9 @@ public final class DependencyTreeGenerator {
                         .map(\.asSource)
                         .joined(separator: " -> "))
                     """
-                }.joined(separator: "\n"))
+                }
+                .sorted()
```

Comment:
we didn't have a test for multiple fulfillable properties! once I added one, it became clear that the error wasn't stable, so we now sort here.